src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -4,18 +4,20 @@ export const adminFeaturesQuerySchema = {
     $id: '#/components/schemas/adminFeaturesQuerySchema',
     type: 'object',
     additionalProperties: true,
+    description: 'Used to filter feature toggles from the admin-api',
     properties: {
         tag: {
             type: 'array',
             items: {
-                type: 'array',
-                items: {
-                    type: 'string',
-                },
+                type: 'string',
             },
+            description:
+                'Used to filter by tags. For each entry, a TAGTYPE:TAGVALUE is expected',
```

Comment:
Is this correct? The tests above use `['some-tag', 'some-other-tag']`. Should they actually be `['tag-type-a:some-tag', 'tag-type-b:some-other-tag']`? If this is correct, then we should encode that into the validator using the `pattern` property. Something like `pattern: '[A-Za-z0-9_-]+:[A-Za-z0-9_-]+'`. And the test above (as it is written today) should fail because the format is incorrect.

We should also add an example to make it easier to understand.

```suggestion
            description:
                'Used to filter by tags. For each entry, a TAGTYPE:TAGVALUE is expected',
             example: ['simple:ui', 'simple:red'],
```